version.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash 
+#     Copyright 2021 Dynatrace LLC
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+echo  "Travis tag     '$TRAVIS_TAG'"
+echo  "Travis commit  '$TRAVIS_COMMIT'"
+
+version=""
+
+if [ -n "$TRAVIS_TAG" ]
```

Comment:
summary from what you explained: 
- these are Travis env vars
- as this version.sh is used only when during "deployment" stage/building the release zip, which is triggered by adding "release.*" tag, the TRAVIS_TAG should be there, and this is just in case